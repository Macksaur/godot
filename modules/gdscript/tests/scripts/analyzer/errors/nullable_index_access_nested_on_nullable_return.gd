class A:
    var foo := A.new()
    func nullable_return() -> String?:
        return null

func test():
    var foobar := A.new()
    foobar.foo.nullable_return()[0]