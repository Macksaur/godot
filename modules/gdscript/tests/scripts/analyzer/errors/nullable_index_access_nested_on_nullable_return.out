GDTEST_ANALYZER_ERROR
Potentially unsafe for the nullable return of "nullable_return()" to be indexed with "0".
